### DISCUSSION TITLE
Github New Version

### DISCUSSION THREAD

**Discussion by diyaryvzz** [community/community#62386] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Is there a new version of Github? If so, can you tell me how can I update it?

**Comment by crizmo** [comment]
Do you mean the website or Github Desktop App ?
if website then there's no updating as such it will always show the latest site

if you mean the Github Desktop App then you can just download the latest version from their site 
or even their app has a inbuilt update feature so you can use that

**Reply by diyaryvzz** [reply]
I enter from the website, I read it in a few places and I heard that there are new things in the update, so I wanted to wonder and ask. Thank you for answering.

**Comment by queenofcorgis** [comment]
@diyaryvzz @crizmo GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
